src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -266,6 +281,11 @@ public Rotation2d getRotation2d() {
   public double getAngleHeading() {
     return MathUtil.angleModulus(m_pigeon.getRotation2d().getRadians());
   }
+
+  public Pigeon2 getPigeon()
+  {
+    return m_pigeon;
+  }
```

Comment:
there is no reason at the momment to access the Peigeon directly, make getter methods for what you need from it